pkg/sdk/validation/manifest/json_remote_type.go

```diff
@@ -29,17 +34,52 @@ func (v *RemoteTypeValidator) Do(ctx context.Context, _ types.ManifestMetadata,
 		return ValidationResult{}, errors.Wrap(err, "while unmarshalling JSON into Type type")
 	}
 
-	var manifestRefsToCheck []hubpublicgraphql.ManifestReference
+	var manifestRefsToCheck []gqlpublicapi.ManifestReference
 
 	// Attributes
 	for path, attr := range entity.Metadata.Attributes {
-		manifestRefsToCheck = append(manifestRefsToCheck, hubpublicgraphql.ManifestReference{
+		manifestRefsToCheck = append(manifestRefsToCheck, gqlpublicapi.ManifestReference{
 			Path:     path,
 			Revision: attr.Revision,
 		})
 	}
 
-	return checkManifestRevisionsExist(ctx, v.hub, manifestRefsToCheck)
+	resExist, err := checkManifestRevisionsExist(ctx, v.hub, manifestRefsToCheck)
+	if err != nil {
+		return ValidationResult{}, err
+	}
+
+	resNodes, err := v.checkAdditionalRefs(ctx, entity)
+	if err != nil {
+		return ValidationResult{}, err
+	}
+
+	return ValidationResult{
+		Errors: append(resExist.Errors, resNodes.Errors...),
+	}, nil
+}
+
+func (v *RemoteTypeValidator) checkAdditionalRefs(ctx context.Context, entity types.Type) (ValidationResult, error) {
+	res := ValidationResult{}
+	if len(entity.Spec.AdditionalRefs) == 0 {
+		return res, nil
+	}
+
+	// AdditionalRefs should point to a concrete path.
+	// It can point only to parent node.
```

Comment:
Not sure what you mean as `concrete path`. Maybe we could say something like:
```suggestion
	// AdditionalRefs should point to an abstract node.
```